pkg/rpcinfo/ctx_test.go

```diff
@@ -49,12 +50,37 @@ func TestGetCtxRPCInfo(t *testing.T) {
 }
 
 func TestPutRPCInfo(t *testing.T) {
-	ri := rpcinfo.NewRPCInfo(nil, nil, nil, nil, nil)
+	var (
+		svcName = "TestServiceName"
+		method  = "TestMethod"
+
+		network = "TestNetWork"
+		addr    = "TestAddress"
+		netAddr = utils.NewNetAddr(network, addr)
+
+		mapKey = "MapTestKey"
+		mapVal = "MapTestValue"
+		tags   = make(map[string]string)
+
+		edp        = rpcinfo.NewEndpointInfo(svcName, method, netAddr, tags)
+		invocation = rpcinfo.NewInvocation(svcName, method)
+		rpcConf    = rpcinfo.NewRPCConfig()
+		rpcStats   = rpcinfo.NewRPCStats()
+	)
+
+	//make sure map has a initial non-null value
+	tags[mapKey] = mapVal
```

Comment:
1. what it is for? no assert below
2. please fix the fmt problem

Reply:
1.That`s for pre-declared variables and initialization.Because the new rpcinfo need non-null fieds to check if the fieds are recycled correctly.
2.That`s my fault.I`ve tried to fix it.

Reply:
> 1. what it is for? no assert below
> 2. please fix the fmt problem
1.That is for pre-declared variables and initialization. Because the test func need rpcinfo have non-null fieds to contrast with fields of recycled rpcinfo.
2.It is my fault.I have tried to fix it.